src/test/java/bankaccount/customer.java

```diff
@@ -0,0 +1,39 @@
+package bankaccount;
+
+import java.util.Scanner;
+
+public class customer {
+    private String name;
+    private int ssn;
+    private String address;
+
+    Scanner scanner = new Scanner(System.in);
+    public customer(String name, int ssn, String address){
+       // System.out.println("Please enter your name");
+        this.name=name;
+        this.ssn=ssn;
+        this.address=address;
+    }
+    //method to edit customer details
+    public void editDetails(){
+        System.out.println("which field you would like to update");
+        System.out.println("A: name");
+        System.out.println("B: address");
+        System.out.println("Please choose A or B");
+        String edit = scanner.next();
+        switch (edit.toUpperCase()){
+            case"A":
+                System.out.println("your name has been changed" + name);
```

Comment:
Looks like you didn't asked to user to provide new name to update existing name.